static/elements/chromedash-feature.html

```diff
@@ -274,20 +270,34 @@
       },
       open: {
         type: Boolean,
-        value: false,
         notify: true,
+        reflectToAttribute: true, // for styling.
+        computed: '_featureOpen(feature.open)',
         observer: '_openChanged'
       }
     },
 
-    toggle: function(e, details) {
-      // Don't toggle panel if tooltip or link is being clicked.
-      var target = Polymer.dom(e).localTarget;
-      if (target.classList.contains('tooltip') || 'tooltip' in target.dataset ||
-          target.tagName == 'A' || target.tagName == 'MULTI-LINKS') {
-        return;
+    _featureOpen: function(open) {
+      return open || false;
+    },
+
+    _toggle: function(e, details) {
+      if (e) {
+        // Don't toggle panel if tooltip or link is being clicked.
+        var target = Polymer.dom(e).localTarget;
+        if (target.classList.contains('tooltip') || 'tooltip' in target.dataset ||
+            target.tagName == 'A' || target.tagName == 'MULTI-LINKS') {
+          return;
+        }
       }
-      this.open = !this.open;
+
+      this.set('feature.open', !this.feature.open);
+
+      this.fire('feature-toggled', {feature: this.feature, open: this.feature.open});
```

Comment:
Added a comment. Basically, `_toggle()` is when a user clicks open a feature card. `_openChanged` only gets invoked when the computed prop `open` changes. That will happen on page load when a feature is deep linked. Firing this event here kicks off the same logic in the main page which does the `history.pushState` etc.